imagestreams/nginx-centos.json

```diff
@@ -50,14 +50,34 @@
       },
       {
         "annotations": {
-          "description": "Build and serve static content via Nginx HTTP server and a reverse proxy (nginx) on RHEL 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/nginx-container/blob/master/1.20/README.md.",
+          "description": "Build and serve static content via Nginx HTTP server and a reverse proxy (nginx) on RHEL 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/nginx-container/blob/master/1.22/README.md.",
           "iconClass": "icon-nginx",
-          "openshift.io/display-name": "Nginx HTTP server and a reverse proxy 1.20 (UBI 8)",
+          "openshift.io/display-name": "Nginx HTTP server and a reverse proxy 1.22 (UBI 9)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
           "sampleRepo": "https://github.com/sclorg/nginx-ex.git",
           "supports": "nginx",
           "tags": "builder,nginx",
-          "version": "1.20"
+          "version": "1.22"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "registry.access.redhat.com/ubi9/nginx-120:latest"
```

Comment:
```suggestion
          "name": "registry.access.redhat.com/ubi9/nginx-122:latest"
```

Reply:
Yay, thanks for spotting, there is the same copy&paste issue a bit more below as well.